release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -536,6 +536,76 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 
 * A new feature has been added to the Console Storage Plug-in that adds Aria labels throughout the installation flow for screen readers. This provides better accessibility for users that use screen readers to access the console.
 
+[id="ocp-4-10-storage-alicloud-disk-csi-operator"]
+==== Persistent storage using the Alibaba AliCloud Disk CSI Driver Operator
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AliCloud Disk. The AliCloud Disk Driver Operator that manages this driver is generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc#persistent-storage-csi-alicloud-disk[AliCloud Disk CSI Driver Operator].
+
+[id="ocp-4-10-storage-azure-file-csi-operator"]
+==== Persistent storage using the Microsoft Azure File CSI Driver Operator (Technology Preview)
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure File. The Azure File Driver Operator that manages this driver is in Technology Preview.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator].
+
+[id="ocp-4-10-storage-ibm-vpc-block-csi-operator"]
+==== Persistent storage using the IBM VPC Block CSI Driver Operator
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for IBM Virtual Private Cloud (VPC) Block. The IBM VPC Block Driver Operator that manages this driver is generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-ibm-vpc-block.adoc#persistent-storage-csi-ibm-vpc-block[IBM VPC Block CSI Driver Operator].
+
+[id="ocp-4-10-storage-vsphere-csi-operator"]
+==== Persistent storage using VMware vSphere CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for vSphere. This feature was previously introduced as a Technology Preview feature in {product-title} 4.8 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[vSphere CSI Driver Operator].
+
+vSphere CSI Driver Operator installation requires:
+
+* vSphere 6.7u3 or later (xref:../installing/installing_vsphere/preparing-to-install-on-vsphere.adoc#installation-vsphere-infrastructure_preparing-to-install-on-vsphere[VMware vSphere CSI Driver Operator requirements])
```

Comment:
@lpettyjo  Should we list all the requirement as same as in https://deploy-preview-38258--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-installation-vsphere-csi?

- Virtual hardware version 15 or greater
- vSphere version 6.7 Update 3 or greater
- VMware ESXi version 6.7 Update 3 or greater

Reply:
others are all good to me.

Reply:
Linked to the spot you indicate.